sdk/core/azure-core/azure/core/messaging.py

```diff
@@ -122,6 +122,16 @@ def from_dict(cls, event):
         :type event: dict
         :rtype: CloudEvent
         """
+        if not all([_ in event for _ in ("source", "type")]):
```

Comment:
Can you add the description of the doc or a link?

Reply:
added https://github.com/cloudevents/spec